tests/vspec/test_node_removal/test_files/units.yaml

```diff
@@ -0,0 +1,22 @@
+# Taken from the units.yaml of VSS catalog
+units:
+  mm:
+    label: millimeter
+    description: Distance measured in millimeters
+    domain: distance
+  m:
+    label: meter
+    description: Distance measured in meters
+    domain: distance
+  degrees/s:
```

Comment:
```suggestion
  degree/s:
```

Reply:
This one is copied from the main units.yaml and I just confirmed that `degrees/s` is correct